op-service/txmgr/cli.go

```diff
@@ -472,11 +478,34 @@ func NewConfig(cfg CLIConfig, l log.Logger) (*Config, error) {
 	res.MinTipCap.Store(minTipCap)
 	res.MaxTipCap.Store(maxTipCap)
 	res.MinBlobTxFee.Store(defaultMinBlobTxFee)
-	res.EnableCellProofs = cfg.EnableCellProofs
 
 	return &res, nil
 }
 
+func fallbackToOsakaCellProofTimeIfKnown(chainID *big.Int, cellProofTime uint64) uint64 {
+	if cellProofTime != math.MaxUint64 {
+		return cellProofTime // We only fallback if nothing is set
+	}
+	if chainID.Cmp(params.MainnetChainConfig.ChainID) == 0 {
+		if params.MainnetChainConfig.OsakaTime == nil {
+			return math.MaxUint64 // not yet scheduled, so we never use cell proofs
+		} else {
+			return *(params.MainnetChainConfig.OsakaTime)
+		}
+	}
+
+	if chainID.Cmp(params.SepoliaChainConfig.ChainID) == 0 {
+		if params.SepoliaChainConfig.OsakaTime == nil {
+			return math.MaxUint64
+		} else {
+			return *(params.SepoliaChainConfig.OsakaTime)
+		}
+	}
```

Comment:
Could shorten this a bit to

```suggestion
	var osakaTime *uint64
	if chainID.Cmp(params.MainnetChainConfig.ChainID) == 0 {
		osakaTime = params.MainnetChainConfig.OsakaTime
	} else if chainID.Cmp(params.SepoliaChainConfig.ChainID) == 0 {
		osakaTime = params.SepoliaChainConfig.OsakaTime
	}
	if osakaTime != nil {
		return *osakaTime
	}
```

Reply:
And do we want to add Holesky here?